Travel Scheduler/Views/SlideMenuUIView.m

```diff
@@ -0,0 +1,90 @@
+//
+//  SlideMenuUIView.m
```

Comment:
This is related to a separate feature. Please don't have spillover code across pull requests!